web/frontend/src/components/slam/index.svelte

```diff
@@ -340,9 +340,22 @@ import { grpc } from '@improbable-eng/grpc-web';
     mapNameError = overrides?.validateMapName(newMapName) ?? '';
   };
 
-  const handleDrop = (event: CustomEvent<string>) => {
-    motionPath = event.detail;
-  };
+  const handleDrop = (event: CustomEvent<string>) => (
+    new Float32Array(
+    event.detail.split('\n').flatMap((str) => {
+      const [xStr, yStr] = str.split(',');
+      if (xStr && yStr) {
+        const x = Number.parseFloat(xStr) / 1000;
+        const y = Number.parseFloat(yStr) / 1000;
+        if (!Number.isNaN(x) && !Number.isNaN(y)) {
+          return [x, y];
+        }
+        return [];
+      }
+      return [];
+    })
+  )
+  );
```

Comment:
This doesn't appear to be assigning the `Float32Array` to `motionPath`, which is needed for the drag-and-drop functionality.

Reply:
🤦 fixed